projectStudyLevelAnnotations.R

```diff
@@ -64,329 +64,357 @@ projects.annotations <- list(
+                       reportedOutcome =c('sleep report',
+                                          'activity report', 'medical history',
+                                          'quality of life', 'demographics survey',
+                                          'research interest', 'family history',
+                                          'height', 'weight'),
+                       digitalAssessmentCategory= c('physical activity', 'cardiovascular',
+                                                    'cognition'),
+                       digitalAssessmentDetails = c('daily physical activity', 'heart rate',
+                                                    '3-minute Psychomotor Vigilance Task (PVT-B)',
+                                                    'tapping', 'step count', 'heart rate',
+                                                    'flights climbed', 'distance',
+                                                    'resting energy burned',
+                                                    'exercise time', 'active energy burned'),
+                       sensorDataType = 'processed',
+                       investigator= c('American Sleep Apnea Association', 'UCSD'),
+                       resourceType = c('curatedData'),
+                       # intervention = '',
+                       keywords = c('sleep'),
+                       dhPortalIndex = 'TRUE',
+                       studyDescriptionLocation = 'syn18492837',
+                       studyDataDescriptionLocation = 'syn18492837/wiki/590797',
+                       studyDescription = 'The SleepHealth Mobile App Study was created to investigate how sleep is associated with general health, daytime alertness, medical conditions, and productivity.',
+                       dataAccessInstructions = 'syn18492837/wiki/590794'),
+
+  ## mPower Study (Rerun Analysis)
+  'syn23277418' = list(consortium = 'mHealth',
+                       collectionType = 'Analysis',
+                       study = 'mPower - Features, model and analysis for Omberg et al (2021)',
+                       dataCollectionMethod= c('active',
+                                               'patient reported outcome',
+                                               'clinical assessment'),
+                       deviceType = c('handheld'),
+                       sensorType = c('touchscreen', 'accelerometer', 'microphone',
+                                      'gyroscope', 'magnetometer'),
+                       devicePlatform = c('iOS'),
+                       deviceLocation = c('hand', 'pocket', 'flat surface'),
+                       diagnosis=  c("Parkinson's Disease", 'control'),
+                       reportedOutcome = c('medication report', 'PDQ-8', 'MDS-UPDRS',
+                                           'demographics survey'),
+                       digitalAssessmentCategory= c('phonation', 'gait',
+                                                    'motor coordination',
```

Comment:
mPower Study (Rerun analysis) syn23277418, is also mentioned in projectProjectLevelAnnotations.R . We should remove repeats.

https://github.com/Sage-Bionetworks/digital-health-portal/blob/231c6c37dcb37f8ba8340e3a5fd12d6d5cc0244d/projectProjectLevelAnnotations.R#L209

Reply:
Artyon is setting the 'motor coordination' annotation at the file level using this file view (syn23545224) and this code https://github.com/Sage-Bionetworks/digital-health-portal/blob/master/Collections%20File%20Level%20Annotations/mPowerRerunFileAnnotations.R

Reply:
Hmm... I'm wondering if someone manually set those at some point? I don't see anywhere where `digitalAssessmentDetails` = "motor coordination" in the code, and yet that's how it was set for those two files. In any case, I've manually corrected the problems.

Reply:
> mPower Study (Rerun analysis) syn23277418, is also mentioned in projectProjectLevelAnnotations.R . We should remove repeats.
> 
> https://github.com/Sage-Bionetworks/digital-health-portal/blob/231c6c37dcb37f8ba8340e3a5fd12d6d5cc0244d/projectProjectLevelAnnotations.R#L209

This may have been my fault. I noticed that mPower Study (Rerun analysis) was put in StudyLevelAnnotations, rather than ProjectLevelAnnotations. I may have added it to Projects, but not deleted it from Studies. I'll take a look and create a new PR.

Reply:
It is a bit convoluted. The file view (syn23545224) for mPower reruns contains a column called analysisType. I suspect they are using this to map the annotations using the file I mentioned above. I could see the file (syn23542620) is of analysisType = 'records per mPower tasks' - which is tagged with `digitalAssessmentDetails` = "motor coordination" . Either way the annotations won't change if we don't run this file again.

Reply:
I'm going to go ahead and merge this PR